server.js

```diff
@@ -1,12 +1,23 @@
 const express = require("express")
 
 const app = express()
+const mongoose = require('mongoose')
+const data ='mongodb+srv://nitinsoni:<Nitin11>@most-searched-web.cc9wsj9.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Move the database connection to the start of the server and use environment variables for sensitive data like the connection string. [important]